ontopy/excelparser.py

```diff
@@ -29,118 +35,248 @@ def create_ontology_from_excel(  # pylint: disable=too-many-arguments
-                            "Missing at least one of the defined parents. "
-                            f"Concept: {name}; Defined parents: {parent_names}"
+                if name in onto:
+                    if not force:
+                        raise ExcelError(
+                            f'Concept "{name}" already in ontology'
                         )
-                        new_loop = False
+                    warnings.warn(
+                        f'Ignoring concept "{name}" since it is already in '
+                        "the ontology."
+                    )
+                    # What to do if we want to add info to this concept?
+                    # Should that be not allowed?
+                    # If it should be allowed the index has to be added to
+                    # added_rows
+                    continue
+
+                if pd.isna(row["subClassOf"]):
+                    if not force:
+                        raise ExcelError(f"{row[0]} has no subClassOf")
+                    parent_names = []  # Should be "owl:Thing"
+                else:
+                    parent_names = str(row["subClassOf"]).split(";")
+
+                parents = []
+                invalid_parent = False
+                for parent_name in parent_names:
+                    try:
+                        parent = onto.get_by_label(parent_name.strip())
+                    except (NoSuchLabelError, ValueError) as exc:
+                        if parent_name not in labels:
+                            if force:
+                                warnings.warn(
+                                    f'Invalid parents for "{name}": '
+                                    f'"{parent_name}".'
+                                )
+                                break
+                            raise ExcelError(
+                                f'Invalid parents for "{name}": {exc}\n'
+                                "Have you forgotten an imported ontology?"
+                            ) from exc
+                        invalid_parent = True
+                        break
                     else:
-                        continue
+                        parents.append(parent)
+
+                if invalid_parent:
+                    continue
+
+                if not parents:
+                    parents = [owlready2.Thing]
 
                 concept = onto.new_entity(name, parents)
+                added_rows.add(index)
                 # Add elucidation
-                _add_literal(
-                    row, concept.elucidation, "Elucidation", only_one=True
-                )
+                try:
+                    _add_literal(
+                        row,
+                        concept.elucidation,
+                        "Elucidation",
+                        only_one=True,
+                    )
+                except AttributeError as err:
+                    if force:
+                        _add_literal(
```

Comment:
Since all `_add_literal`s seem to be wrapped in a `try/except` - might this also fail?